recipe/install-rust-src.sh

```diff
@@ -4,9 +4,11 @@ set -ex
 
 cd rust-src
 
+DESTDIR=/
+
 ./install.sh --prefix="$PREFIX" --destdir="$DESTDIR"
 
-rm "${PREFIX}"/lib/rustlib/components
-rm "${PREFIX}"/lib/rustlib/install.log
-rm "${PREFIX}"/lib/rustlib/rust-installer-version
-rm "${PREFIX}"/lib/rustlib/uninstall.sh
+rm "${DESTDIR}"/"${PREFIX}"/lib/rustlib/components
+rm "${DESTDIR}"/"${PREFIX}"/lib/rustlib/install.log
+rm "${DESTDIR}"/"${PREFIX}"/lib/rustlib/rust-installer-version
+rm "${DESTDIR}"/"${PREFIX}"/lib/rustlib/uninstall.sh
```

Comment:
Are the changes to this file necessary? If we are just setting `$DESTDIR` to `/`, it seems to me that this file does not need to be modified.

Reply:
Yes, these changes are not necessary. But I think it's a better implementation because if you want to change DESTDIR and this is good to go.